pkg/operator/ceph/config/monstore.go

```diff
@@ -207,3 +191,64 @@ func (m *MonStore) SetKeyValue(key, value string) error {
 	}
 	return nil
 }
+
+func (m *MonStore) SetAll(clientName string, settings map[string]string) error {
+	assimilateConfPath, err := ioutil.TempFile("", "")
```

Comment:
ioutil functions are mostly deprecated started go 1.16. The [doc](https://pkg.go.dev/io/ioutil@go1.18) is suggesting to use the `os` or `io` package.

Reply:
This code just moved, but good point to clean it up at the same time, thanks